umh.docs.umh.app/content/en/docs/features/datainfrastructure/unified-namespace.md

```diff
@@ -1,67 +1,114 @@
+[Node-RED](/docs/architecture/data-infrastructure/unified-namespace/node-red/)
+or [Benthos UMH](/docs/features/connectivity/benthos-umh/). This step involves
+adjusting payloads and topics as needed. It's also important to understand and
+follow the ISA95 standard model for data organization, using JSON as the
+primary format.
+
+Additionally, the [Data Bridge](/docs/architecture/data-infrastructure/unified-namespace/data-bridge/)
+microservice plays a crucial role in transferring and transforming data between
+MQTT and Kafka, ensuring that it adheres to the UMH data model. You can
+configure a merge point to consolidate messages from multiple MQTT topics into
+a single Kafka topic. For instance, if you set a merge point of 3, the Data
+Bridge will consolidate messages from more detailed topics like
+`umh/v1/plant1/machineA/temperature` into a broader topic like `umh/v1/plant1`.
+This process helps in organizing and managing data efficiently, ensuring that
+messages are grouped logically while retaining key information for each topic
+in the Kafka message key.
 
 {{% notice tip %}}
-**Recommendation:** Send messages from IoT devices via MQTT and then work in Kafka only.
+**Recommendation:** Send messages from IoT devices via MQTT and then work in
+Kafka only.
 {{% /notice %}}
 
 ## What are the limitations?
 
-- Messages are only bridged between MQTT and Kafka if they fulfill the following requirements:
-  - payload is a valid JSON OR message is sent to the `ia/raw` topic
-  - only sent to topics matching the [allowed topics in the UMH datamodel](/docs/architecture/datamodel/messages/), independent of what is configured in the environment variables (will be changed soon)
-  - The topic lengths can be maximum 249 characters as this is a Kafka limitation
-  - Only the following characters are allowed in the topic: `a-z`, `A-Z`, `_` and `-`
-  - Max. messages size for the mqtt-kafka-bridge is 0.95MB (1000000 bytes). If you have more, we recommend using Kafka directly and not bridging it via MQTT.
-- Messages from MQTT to Kafka will be published under a different topic:
-  - Spaces will be removed
-  - `/` characters will be replaced with a `.`
-  - and vice versa
-- By default, there will be no Authorization and Authentication on the MQTT broker. [You need to enable authentication and authorization yourself](/docs/production-guide/security/).
-- The MQTT or Kafka broker is not exposed externally by default. [You need to enable external MQTT access first](/docs/production-guide/administration/access-mqtt-outside-cluster/), or alternatively [expose Kafka externally](/docs/production-guide/administration/access-kafka-outside-cluster/).
+The limitations of the Unified Namespace primarily revolve around its technical
+complexity. Integrating MQTT with Kafka can be challenging, requiring a solid
+understanding of both systems. Additionally, the specific topic structure
+mandated by UMH must be adhered to, which might limit flexibility in some
```

Comment:
even though it is specific, there is still the _schema which allows to do whatever one wants (as long as the first parts of the topic remain the same).